app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -16,7 +17,7 @@ Environment="ETCD_IMAGE_TAG=@ETCD_IMAGE_TAG@"
 Environment="ETCD_NAME=%m"
 Environment="ETCD_USER=etcd"
 Environment="ETCD_DATA_DIR=/var/lib/etcd"
-Environment="RKT_RUN_ARGS=--uuid-file-save=/var/lib/coreos/etcd-member-wrapper.uuid"
+Environment="RKT_RUN_ARGS=--uuid-file-save=/var/lib/coreos/etcd-member-wrapper.uuid --dns=host"
```

Comment:
If we want to change this by default it should be in the wrapper script, not in the service.

I also don't think we should change it because this flag is deceptive; for stage1-fly it operates almost the same as the default option (bindmount vs copy).
Including this flag implies it does something, when the reality is I don't think it is effectively different in any notable way.

Reply:
I've pulled the --dns=host because it doesn't do what I expected here. 

In my testing I added the arg and then changed the resolvers that were provided by dhcp. I then waited for the new lease to pick up the resolvers. When `--dns=host` is configured I see systemd-resolved complaining with:
```
Oct 19 18:18:12 node001.metal.k8s.work systemd-resolved[9442]: Failed to move private resolv.conf file into place: Device or resource busy
Oct 19 18:18:12 node001.metal.k8s.work systemd-resolved[9442]: Failed to move private resolv.conf file into place: Device or resource busy
Oct 19 18:18:12 node001.metal.k8s.work systemd-resolved[9442]: Failed to move private resolv.conf file into place: Device or resource busy
Oct 19 18:18:12 node001.metal.k8s.work systemd-resolved[9442]: Failed to move private resolv.conf file into place: Device or resource busy
Oct 19 18:18:12 node001.metal.k8s.work systemd-resolved[9442]: Failed to move private resolv.conf file into place: Device or resource busy
```